### PR TITLE
NEUBURG-13: Format Mail Bodies

### PR DESCRIPTION
**Pull request by steffenkleinle**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -22,11 +22,13 @@
...
     "express-validator": "^5.0.2",
     "http-status-codes": "1.3.0",
+    "moment": "^2.22.1",
     "mongodb": "^3.0.4",
     "mongoose": "^5.0.11",
     "morgan": "^1.9.0",
     "nodemailer": "^4.6.0",
-    "pug": "^2.0.0-rc.4"
+    "pug": "^2.0.3",
+    "pug-cli": "^1.0.0-alpha6"
```
Comment (maxammann) [inline package.json:31]: Can we use the stable version here? Or would this break the existing code?

**Review by maxammann — APPROVED** [review]

File: src/services/MailService.js [hunk 2]
```diff
@@ -3,17 +3,23 @@
...
+const renderRequestConfirmationMail = compileFile('src/views/requestConfirmationMail.pug')
 const renderDeletionMail = compileFile('src/views/deletionMail.pug')
 const renderExtensionMail = compileFile('src/views/extensionMail.pug')
 
-const getConfirmationUrl = (city: string, token: string): string => `http://neuburg.wohnen.integreat-app.de/offer/${token}/confirm`
-const getDeletionUrl = (city: string, token: string): string => `http://neuburg.wohnen.integreat-app.de/offer/${token}/delete`
-const getExtensionUrl = (city: string, token: string): string => `http://neuburg.wohnen.integreat-app.de/offer/${token}/extend`
+const getConfirmationUrl = (city: string, token: string): string => `http://api.wohnen.integreat-app.de/v0/${city}/${token}/confirm`
+const getExtensionUrl = (city: string, token: string): string => `http://api.wohnen.integreat-app.de/v0/${city}/${token}/extend`
+const getDeletionUrl = (city: string, token: string): string => `http://api.wohnen.integreat-app.de/v0/${city}/${token}`
```
Comment (michael-markl) [inline src/services/MailService.js]: These are not the correct urls. These lead to the API, whereas they should lead to the frontend.
The correct urls would be:
```js
const getConfirmationUrl = (city: string, token: string): string => `https://${city.hostname}/offer/${token}/confirm`
const getExtensionUrl = (city: string, token: string): string => `https://${city.hostname}/offer/${token}/extend`
const getDeletionUrl = (city: string, token: string): string => `https://${city.hostname}/offer/${token}/delete`
```

**Review by michael-markl — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (steffenkleinle) [reply package.json:31]: I didn't check, I just used this just to have previews of the html generated by the pug email files in intelliJ, we could also remove it again

File: src/services/MailService.js (hunk 2 above)
Comment (steffenkleinle) [reply src/services/MailService.js]: Done

**Review by michael-markl — APPROVED** [review]

**Pull request closed without merging**